client/lib/tree-convert/test/tree-convert.js

```diff
@@ -1,69 +1,73 @@
-global.localStorage = require( 'localStorage' );
-
-var expect = require( 'chai' ).expect,
-	cloneDeep = require( 'lodash/cloneDeep' ),
-	TreeConvert = require( 'lib/tree-convert' ),
-	fixtures = require( './fixtures' );
-
-
-describe( 'TreeConvert', function() {
-	beforeEach( function() {
-		this.converter = TreeConvert();
-		this.menu = cloneDeep( fixtures.menusFlat.menus[0] );
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import cloneDeep from 'lodash/cloneDeep';
+
+/**
+ * Internal dependencies
+ */
+import { menusFlat, menuBadParent } from './fixtures';
+import TreeConvert from '../index';
+
+describe( 'TreeConvert', () => {
+	let converter, menu;
+
+	beforeEach( () => {
+		converter = TreeConvert();
+		menu = cloneDeep( menusFlat.menus[0] );
 	} );
 
-	describe( 'treeify', function() {
-		it( 'should turn a flat parent/child structure into a tree', function() {
-			var treeified = this.converter.treeify( this.menu.items );
+	describe( 'treeify', () => {
+		it( 'should turn a flat parent/child structure into a tree', () => {
+			const treeified = converter.treeify( menu.items );
 
 			expect( treeified ).to.have.length( 3 );
-			expect( treeified[2].items ).to.be.instanceOf(Array);
+			expect( treeified[2].items ).to.be.instanceOf( Array );
 		} );
 
-		it( 'should order sibling items by their "order" property', function() {
-			var treeified = this.converter.treeify( this.menu.items );
+		it( 'should order sibling items by their "order" property', () => {
+			const treeified = converter.treeify( menu.items );
 
 			expect( treeified[1].items[0] ).to.have.property( 'name', 'Socks' );
 		} );
 
-		it( 'should remove the "order" property from items, since order will be implicit', function() {
-			var treeified = this.converter.treeify( this.menu.items );
+		it( 'should remove the "order" property from items, since order will be implicit', () => {
+			const treeified = converter.treeify( menu.items );
 
 			expect( treeified[1].items[0] ).not.to.have.property( 'order' );
 		} );
 
-		it( 'should handle bad parent data, falling back to parent 0', function() {
-			var menu = cloneDeep( fixtures.menuBadParent ),
-				treeified = this.converter.treeify( menu.items );
+		it( 'should handle bad parent data, falling back to parent 0', () => {
+			const badMenu = cloneDeep( menuBadParent );
```

Comment:
I must have explain badly how we deal with `const`. `let` and `var` :)

We usually use one `const` declaration for all variables in given scope. Exception would the case when you want to use it after `if` statement or inside nested block like:
```js
const a = 1, b = 2, c = 3;
let i = 0;
while ( i < c ) {
   let valid = true;
   if (  i < b) {
     valid = false;
   }
   i = i +2;
}

for ( let j = 0; j < i; j++ ) {
  const sth = 'sth';
}
```
This code most likely doesn't make any sense. I just wanted to present the idea :)